src/no/runsafe/UserControl/database/PlayerKickLog.java

```diff
@@ -22,22 +25,39 @@ public ISchemaUpdate getSchemaUpdateQueries()
 		update.addQueries(
 			"CREATE TABLE player_kick_log (" +
 				"`name` varchar(255) NOT NULL," +
-				"`timestamp` datetime NULL," +
+				"`timestamp` datetime NOT NULL," +

```

Comment:
timestamp doesn't appear to be set as null on the test or the live server, so I'm guessing it might have been created manually since I couldn't find a commit that changed it from not null to null.